src/api/.rubocop_todo.yml

```diff
@@ -908,85 +908,7 @@ RSpec/MultipleMemoizedHelpers:
-    - 'spec/controllers/webui/project_controller_spec.rb'
-    - 'spec/controllers/webui/projects/meta_controller_spec.rb'
-    - 'spec/controllers/webui/projects/rebuild_times_controller_spec.rb'
-    - 'spec/controllers/webui/repositories_controller_spec.rb'
-    - 'spec/controllers/webui/request_controller_spec.rb'
-    - 'spec/controllers/webui/search_controller_spec.rb'
-    - 'spec/controllers/webui/session_controller_spec.rb'
-    - 'spec/controllers/webui/staging/excluded_requests_controller_spec.rb'
-    - 'spec/controllers/webui/users_controller_spec.rb'
-    - 'spec/db/data/regenerate_notifications_spec.rb'
-    - 'spec/features/beta/webui/attributes_spec.rb'
-    - 'spec/features/beta/webui/requests_spec.rb'
-    - 'spec/features/webui/attributes_spec.rb'
-    - 'spec/features/webui/requests_spec.rb'
-    - 'spec/helpers/event_mailer_helper_spec.rb'
-    - 'spec/helpers/webui/package_helper_spec.rb'
-    - 'spec/helpers/webui/request_helper_spec.rb'
-    - 'spec/helpers/webui/webui_helper_spec.rb'
-    - 'spec/jobs/staging_project_accept_job_spec.rb'
-    - 'spec/lib/authenticator_spec.rb'
-    - 'spec/mailers/event_mailer_spec.rb'
-    - 'spec/mixins/build_log_support_spec.rb'
-    - 'spec/models/attrib_spec.rb'
-    - 'spec/models/backend/file_spec.rb'
-    - 'spec/models/branch_package_spec.rb'
-    - 'spec/models/bs_request/find_for/group_spec.rb'
-    - 'spec/models/bs_request/find_for/user_spec.rb'
-    - 'spec/models/bs_request_action/differ/for_source_spec.rb'
-    - 'spec/models/bs_request_action/differ/query_builder_for_superseded_spec.rb'
-    - 'spec/models/bs_request_action/differ/source_package_finder_spec.rb'
-    - 'spec/models/bs_request_action_spec.rb'
-    - 'spec/models/bs_request_spec.rb'
-    - 'spec/models/cloud/backend/upload_job_spec.rb'
-    - 'spec/models/configuration_spec.rb'
-    - 'spec/models/event_subscription/find_for_event_spec.rb'
-    - 'spec/models/event_subscription/for_role_form_spec.rb'
-    - 'spec/models/flag/specified_flags_spec.rb'
-    - 'spec/models/full_text_search_spec.rb'
-    - 'spec/models/issue_tracker/issue_summary_spec.rb'
-    - 'spec/models/kiwi/image_spec.rb'
-    - 'spec/models/kiwi/repository_spec.rb'
-    - 'spec/models/local_build_result/for_package_spec.rb'
-    - 'spec/models/owner_search_spec.rb'
-    - 'spec/models/package_spec.rb'
-    - 'spec/models/project/image_template_spec.rb'
-    - 'spec/models/project/staging_project_spec.rb'
-    - 'spec/models/project/update_from_xml_command_spec.rb'
-    - 'spec/models/project_spec.rb'
-    - 'spec/models/project_status/pack_info_spec.rb'
-    - 'spec/models/repository_spec.rb'
-    - 'spec/models/status_message_spec.rb'
-    - 'spec/models/unregistered_user_spec.rb'
-    - 'spec/models/user_ldap_strategy_spec.rb'
-    - 'spec/models/user_spec.rb'
-    - 'spec/requests/kerberos_login_spec.rb'
-    - 'spec/services/staging/staging_project_creator_spec.rb'
+  Enabled: false
```

Comment:
If we decide to disable this cop, it should be done in the main RuboCop configuration file (`src/api/.rubocop.yml`).